scipy/stats/stats.py

```diff
@@ -3692,6 +3694,198 @@ def f_oneway(*args, axis=0):
+        The computed A statistic of the test.
+    pvalue : float
+        The associated p-value from the chi-squared distribution.
+    References
+        ----------
+        ... [1] Alexander, Ralph A., and Diane M. Govern. “A New and Simpler
+                Approximation for ANOVA under Variance Heterogeneity.” Journal
+                of Educational Statistics, vol. 19, no. 2, 1994, pp. 91–101.
+                JSTOR, www.jstor.org/stable/1165140. Accessed 12 Sept. 2020.
+        ... [2] Kingsley Ochuko, T., Abdullah, S., Binti Zain, Z., & Soaad Syed
+                Yahaya, S. (2015). The Modification and Evaluation of the
+                Alexander-Govern Test in Terms of Power. Modern Applied
+                Science, 9(13), 1. https://doi.org/10.5539/mas.v9n13p1
+    Examples
+        --------
+        >>> from scipy.stats import alexandergovern_alt, Kruskal, F_oneway
+        >>> young = [482.43, 484.36, 488.84, 495.15, 495.24, 502.69, 504.62,
+        ...         518.29, 519.1, 524.1, 524.12, 531.18, 548.42, 572.1,
+        ...         584.68, 609.09, 609.53, 666.63, 676.4]
+        >>> middle = [335.59, 338.43, 353.54, 404.27, 437.5, 469.01, 485.85,
+        ...          487.3, 493.08, 494.31, 499.1, 800]
+        >>> old = [519.01, 528.5, 530.23, 536.03, 538.56, 538.83, 557.24,
+        ...        558.61, 558.95, 565.43, 586.39, 594.69, 629.22, 645.69,
+        ...        691.84]
+        >>> A, p = alexandergovern_alt(young, middle, old)
+        >>> A, p
+        (6.941146076872535, 0.03109920451449096)
+    """
+
+    def calc_sem(data, mean, length):
+        """Calculates standard error using precalculated values"""
+        sum_ = np.sum((data - mean) ** 2)
+        d = length * (length - 1)
+        return (sum_ / d) ** .5
+
+    def calc_z2(var_w, sem, mean, length):
+        """Calculates the AG Z^2 statistic for the passed array"""
+        # Calc degrees of freedom and t, as described in formula [2](7)
+        df = length - 1
+        t = (mean - var_w) / sem
+
+        # Calculate a,b,c using evaluating AG's formula [2](9), [2](10)
+        a = df - .5
+        b = 48 * a ** 2
+        c = (a * np.log(1 + t ** 2 / df)) ** .5
+
+        # Calculate the three terms in evaluating AG's Z formula [2](8)
+        t0 = c
+        t1 = (c ** 3 + 3 * c) / b
+        t3 = (4 * c ** 7 + 33 * c ** 5 + 240 * c ** 3 + 855 * c)
+        t3d = (10 * b ** 2 + 8 * b * c ** 4 + 1000 * b)
+
+        return (t0 + t1 - t3/t3d) ** 2
+
+    if len(args) < 2:
+        raise TypeError(f"2 or more inputs required, got {len(args)}")
+
+    # Convert passed list to ndarray. `fromiter` is faster than `asarray`.
+    a = [np.fromiter(A, float) for A in args]
+
+    # Calculate sample constants to avoid recalculating
+    lens = np.asarray([len(A) for A in a])
+    means = np.asarray([np.mean(A) for A in a])
+    # List comprehension used over vectorization to avoid broadcasting errors
+    standard_error = np.asarray([calc_sem(a, m, l) for a, m, l
```

Comment:
```suggestion
    standard_error = np.asarray([np.linalg.norm(a - m)/np.sqrt(l) for a, m, l
```
Or something similar